compiler-plugin/src/main/kotlin/arrow/meta/plugins/proofs/phases/resolve/ProofResolution.kt

```diff
@@ -148,7 +148,9 @@ fun FunctionDescriptor.asProof(): Proof? =
   extensionReceiverParameter?.type?.let { from ->
     returnType?.let { to ->
       val annotationArgs = annotations.first().allValueArguments
-      val value: ConstantValue<*>? = annotationArgs[Name.identifier("of")]
+      val value: ConstantValue<*> = annotationArgs.getValue(Name.identifier("of"))
```

Comment:
I'm totally agnostic on this, so please let me know if this doesn't make sense, but. Are we sure this value argument for the annotation is always gonna be there? we can call this function for any function descriptor, not just for type proofs ones.

Reply:
You're actually right 👍 Dani just changed that now :)

Reply:
Yeah, actually I was seeing errors in the logs because of that..